parsing/builtin_attributes.mli

```diff
@@ -61,6 +66,10 @@ val mark_alerts_used : Parsetree.attributes -> unit
     attributes into the environment. *)
 val mark_warn_on_literal_pattern_used : Parsetree.attributes -> unit
 
+(** Marks the attributes hiding in the payload of another attribute (or
+    extension node) used, for the purposes of misplaced attribute warnings. *)
```

Comment:
A natural question is: why have this function? The answer would be in the Note. :grin: